tests/Native/Test/Text.js

```diff
@@ -0,0 +1,23 @@
+Elm.Native.Test      = Elm.Native.Test || {};
+Elm.Native.Test.Text = {};
+Elm.Native.Test.Text.make = function(localRuntime) {
+    localRuntime.Native           = localRuntime.Native           || {};
+    localRuntime.Native.Test      = localRuntime.Native.Test      || {};
+    localRuntime.Native.Test.Text = localRuntime.Native.Test.Text || {};
```

Comment:
Don't do this aligning stuff. It's really frail. When any line changes, lots of things need to change. Not worth it in the long run!